improver/__init__.py

```diff
@@ -31,6 +31,8 @@
 """Module containing plugin base class."""
 from abc import ABC, abstractmethod
 
+__version__ = "0.14.1"
```

Comment:
An alternative to defining the version here is to do it in the setup.cfg file. Then here you can do the following:

```python
import importlib_metadata
__version__ = importlib_metadata.version('improver')
```

Reply:
I have no strong opinion either day, WDYT @tjtg ?

Reply:
I don't have strong opinions about this, so happy to leave it as is.